### ISSUE TITLE
Adding 'description' to Account/General tab PHP Warning

### ISSUE THREAD

**Issue by MissVeronica** [ultimatemember/ultimatemember#1316]
**Describe the bug**
Warning: Undefined array key “mode” in .../wp-content/plugins/ultimate-member/includes/core/class-fields.php on line 2783

**To Reproduce**
```
add_filter( 'um_account_tab_general_fields', 'um_account_tab_general_fields_description', 10, 1 );
function um_account_tab_general_fields_description( $args ) {

    if ( ! strpos( $args, ',description' )) {
	    $args = str_replace( ',single_user_password', ',description,single_user_password', $args );
    }
	return $args;
}
```

**Expected behavior**
Test in `class-fields.php` on line 2783 `$this->global_args['mode']` being set and not empty.

**UM Support Forum issue**
https://wordpress.org/support/topic/how-to-add-custom-field-into-general-tab-in-account-page/

**Comment by BlackStar1991** [comment]
I see this problem on a account page if add field 'description'. 
```
function um_account_tab_general_fields( $args, $shortcode_args ) {

    // Fields Meta Keys.
    $args .= ',description';
    $args .= ',country';
    $args .= ',city';

    return $args;
}
add_filter( 'um_account_tab_general_fields', 'um_account_tab_general_fields', 10, 2 );
```
I see this problem for users with Role Administrator
`Warning: Undefined array key "mode" in D:\LocalProjects\ausuppreview-dev\app\public\wp-content\plugins\ultimate-member\includes\core\class-fields.php on line 2782`
It's still there  UM version 2.6.11

**Comment by nikitasinelnikov** [comment]
Hi @BlackStar1991 

There will be a fix in 2.6.12 version. You can check the development/2.6.12 branch.

We are planning to release 2.6.12 version on the next week.

Best Regards!
